events/Lambda/Invoke.json

```diff
@@ -10,6 +10,43 @@
     "mitreAttackTechniques": [
         "T1496 - Resource Hijacking"
     ],
+    "mitreAttackSubTechniques": [
+
+    ],
+    "unverifiedMitreAttackTechniques": [
+        {
+            "technique": "T1059 - Command and Scripting Interpreter",
+            "reason": "Lambda functions can be used to execute scripts and commands, allowing attackers to run arbitrary code within the AWS environment."
+        },
+        {
+            "technique": "T1071 - Application Layer Protocol",
+            "reason": "The Invoke API call can be used to establish communication channels over various application layer protocols for command and control purposes."
+        },
+        {
+            "technique": "T1036 - Masquerading",
+            "reason": "Attackers can invoke Lambda functions under the guise of legitimate requests to evade detection."
+        },
+        {
+            "technique": "T1105 - Ingress Tool Transfer",
+            "reason": "An attacker can use Lambda functions to download or transfer malicious tools into the environment."
+        },
+        {
+            "technique": "T1074 - Data Staged",
+            "reason": "Lambda functions can be used to stage data for exfiltration, storing collected information temporarily."
+        },
+        {
+            "technique": "T1219 - Remote Access Software",
+            "reason": "Attackers can use Lambda functions as a form of remote access to maintain control over compromised systems."
+        },
+        {
+            "technique": "T1027 - Obfuscated Files or Information",
+            "reason": "Attackers might use obfuscation techniques in Lambda functions to hide malicious activities."
```

Comment:
The same I tought Exploit Public-Facing Application seems more invoke, I think obfuscate Files or Information seems more CreateFunction as is there when you can create a lambda with obfuscated malicious actions.

Reply:
Swapped them.

Reply:
I hhave also included serverless execution as a new technique.